### PR TITLE
added Code of Linear Search in a Recursive way

### PR DESCRIPTION
**Pull request by 007AnupamSharma**
# Description
it is defined as a sequential search algorithm that starts at one end and goes through each element of a list until the desired element is found, otherwise the search continues till the end of the data set.

Linear search works by comparing each element of the data structure with the key to be found. 

LinearSearch (array, index, key):
    if index < 0:
        return -1;
    if item = key:
        return index
    return LinearSearch (array, index-1, key)

### CODE REVIEW

**Comment by 007AnupamSharma** [comment]
@kelvins please review the code

**Review by kelvins — CHANGES_REQUESTED** [review]
Thanks @007AnupamSharma! Please, add the `LinearSearchRecursive.cpp` link to the README file.

File: src/cpp/LinearSearchRecursive.exe
[the remark this answers is not in the document]
Comment (007AnupamSharma) [reply src/cpp/LinearSearchRecursive.exe:1]: ok @kelvins

File: src/cpp/LinearSearchRecursive.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include <vector>
+using namespace std;
+
+bool  linearSearchRecursive(int arr[], int n, int index, int target){
+    if(index >= n){
+        return false;
+    }
+
+    if(arr[index] == target){
+        return true;
```
[the remark this answers is not in the document]
Comment (007AnupamSharma) [reply src/cpp/LinearSearchRecursive.cpp:11]: @kelvins Ok sir

File: src/cpp/LinearSearchRecursive.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+bool  linearSearchRecursive(int arr[], int n, int index, int target){
```
[the remark this answers is not in the document]
Comment (007AnupamSharma) [reply src/cpp/LinearSearchRecursive.cpp:5]: ya Done

**Comment by 007AnupamSharma** [comment]
@kelvins i had made all changes that you want ... would you please review it

**Review by kelvins — APPROVED** [review]

**Pull request merged**